hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -212,8 +210,7 @@
         { bits: "24",
           name: "RXEMPTY",
           desc: '''When high, indicates that the receive fifo is empty.
-                   Any reads from RX FIFO will cause an error interrupt.
-                   '''
+                   Any reads from RX FIFO will cause an error interrupt.'''
```

Comment:
What was the motivation for these changes?

The [Hjson spec](https://hjson.github.io/syntax.html#multiline-strings) is a little hard to understand on multiline strings, but I think formatting like this is safest:

```
desc:
  '''
  When high, indicates that the receive fifo is empty.
  Any reads from RX FIFO will cause an error interrupt.
  '''
```

Regardless, if this works then go ahead with it - we can get something more stable in the future with a Hjson linter.

Reply:
Also, it seems we're on RTL lockdown so make sure to tag someone who can override the block or defer the Hjson changes

Reply:
I tried to make the minimal changes required to remove the whitespaces. I agree, the triple quotes on their own line is nicer.